task/aws/resources/resource_launch_template.go

```diff
@@ -110,6 +109,10 @@ func (l *LaunchTemplate) Create(ctx context.Context) error {
 		},
 	}
 
+	if size := l.Attributes.Size.Storage; size > 0 {
+		input.LaunchTemplateData.BlockDeviceMappings[0].Ebs.VolumeSize = aws.Int32(int32(size))
+	}
```

Comment:
Why? Is there any fundamental difference? To avoid 0?

Reply:
Unset means automatic

Reply:
Should that end up with different sizes depending on the provider?

Reply:
Yes, providers calculate those automatically from the base image